diff -ur libdvdread-6.0.1-orig/msvc/msvc/contrib/dlfcn.c libdvdread-6.0.1/msvc/msvc/contrib/dlfcn.c
--- libdvdread-6.0.1-orig/msvc/msvc/contrib/dlfcn.c	2019-03-06 09:33:41.575001000 -0800
+++ libdvdread-6.0.1/msvc/msvc/contrib/dlfcn.c	2019-03-06 09:34:58.251001000 -0800
@@ -27,7 +27,7 @@
      */
     (void)strncpy(path, module_name, MAX_PATH);
     p = path;
-    while (p = strchr(p, '/'))
+    while ((p = strchr(p, '/')))
         *p = '\\';
 
     /* First assume the dso/dll's required by -this- dso are sitting in the
